controllers/cruisecontroltask_controller.go

```diff
@@ -226,6 +226,10 @@ func SetupCruiseControlWithManager(mgr ctrl.Manager) *ctrl.Builder {
 				if err != nil {
 					return false
 				}
+				// Skip object if v1alpha1.OwnershipAnnotation is set as it is owned by other system.
+				if ok := ObjectManagedByClusterRegistry(object); ok {
```

Comment:
I believe this filtering has to be added to `CreateFunc` as well to avoid triggering a reconcile when a `KafkaCluster` owned by cluster registry is created initially. The same for `DeleteFunc`

Reply:
Alright. Added these.